CHANGELOG.md

```diff
@@ -10,9 +10,16 @@ Every Message starts with one of the following words:
  - `Fix`
  - `Switch` internal change
  - `Remove` functionality
+ - `Update`
 
 ## [Unreleased]
 
+## [0.3.2] - 2017-02-12
+
+ - Add systemd service unit `coredumpbot.service`
+ - Add new category `Update`
```

Comment:
I don't think you need to add change to the changelog format to the version changelog :)